x-pack/plugins/cross_cluster_replication/public/app/sections/home/follower_indices_list/components/detail_panel/detail_panel.js

```diff
@@ -104,6 +167,180 @@ export class DetailPanelUi extends Component {
               </EuiFlexItem>
             </EuiFlexGroup>
 
+            {isPaused ? null : (
+              <Fragment>
+                <EuiSpacer size="s" />
+
+                <EuiFlexGroup>
+                  <EuiFlexItem>
+                    <EuiDescriptionListTitle>
+                      <EuiTitle size="xs">
+                        <FormattedMessage
```

Comment:
Small note: this is a lot of duplication of translations with the form fields. Unless we don't care at all about the size of bytes sent to the client, we should be able to reuse those translations. It seems that we don't care too much about the size of our `.bundle.js` so maybe it doesn't matter but wanted to pointed it out. Thoughts @cjcenizal @bmcconaghy ?

Reply:
If we're using the same string everywhere in the same context, e.g. "Max read request operation count" as a label for a section or field, then yes I agree. I think we could safely refer to a single instance of each string in multiple places without interfering with the translators' work. I also think it could be easy to follow while reading the code if we name the variables well.

Reply:
I will change these labels so they use the same ones from form